main.tf

```diff
@@ -112,7 +117,7 @@ locals {
 
 module "gke_app" {
   source  = "wandb/wandb/kubernetes"
-  version = "1.1.1"
+  version = "1.1.2"
```

Comment:
Is there a version 1.1.2 published yet? https://github.com/wandb/terraform-kubernetes-wandb/releases

Reply:
It will be after https://github.com/wandb/terraform-kubernetes-wandb/pull/12